dts/arm/realtek/ec/rts5912.dtsi

```diff
@@ -63,6 +63,13 @@
 			reg = <0x40090000 0x300>;
 		};
 	};
+
+	swj_port: swj-port {
+		compatible = "swj-connector";
+		pinctrl-0 = <	&jtag_tdi_gpio87 &jtag_tdo_gpio88 &jtag_rst_gpio89
+				&jtag_clk_gpio90 &jtag_tms_gpio91>;
```

Comment:
nit, the indentation got a bit funny here, drop the leading spaces

Reply:
We remove the leading spaces in next version of PR and align `jtag_clk_gpio90` to the `jtag_tdi_gpio87.`